Services/CO.CDP.Forms.WebApi/UseCase/UpdateFormSectionAnswersUseCase.cs

```diff
@@ -111,54 +87,64 @@ private static void ValidateQuestions(List<FormAnswer> answers, Dictionary<Guid,
         }
     }
 
-    private List<Persistence.FormAnswer> MapAnswers(
+    private ICollection<Persistence.FormAnswer> MapAnswers(
         List<FormAnswer> answers,
         Dictionary<Guid, Persistence.FormQuestion> questionDictionary,
         ICollection<Persistence.FormAnswer> existingAnswers)
     {
-        List<Persistence.FormAnswer> answersList = [];
+        // first, include answers that were already there and update them if necessary
+        var answersList = existingAnswers
+            .Select(ea => UpdateAnswerValues(ea, answers.FirstOrDefault(a => a.Id == ea.Guid)));
+
+        // next, include new answers making sure to filter updates to the existing ones
```

Comment:
Please remove comments, do we need comments to be checked in for this PR

Reply:
@JBaigGoaco I guess, it gives more code readability.

Reply:
I'm first to remove comments, but in this case I thought they explain the context.